op-chain-ops/deployer/pipeline/implementations.go

```diff
@@ -77,7 +77,7 @@ func DeployImplementations(ctx context.Context, env *Env, intent *state.Intent,
 	}
 
 	st.ImplementationsDeployment = &state.ImplementationsDeployment{
-		OpsmAddress:                             dio.Opsm,
+		OpsmAddress:                             dio.OpsmProxy,
```

Comment:
[nit] should be renamed to `OpsmProxyAddress` if it's pointing at a proxy.

Reply:
@mslipper you're suggesting this right?
```suggestion
		OpsmProxyAddress:                             dio.OpsmProxy,
```

Reply:
Updated in: https://github.com/ethereum-optimism/optimism/pull/11972/commits/3d9e6517231ee34beb036e5c561ff6e6ca4f64df